### ISSUE TITLE
Upcoming Changes to the Streams v5 Endpoints

### ISSUE THREAD

**Issue by BarryCarlyon** [twitchdev/issues#64]
**Brief description**

Documentation and example responses need to be updated to account for the remove `_total` and changes to offset/limits

**Additional context or questions**

https://discuss.dev.twitch.tv/t/upcoming-changes-to-the-streams-v5-endpoints/23926
https://discuss.dev.twitch.tv/t/get-followed-streams-has-been-changed/24200

**Comment by oliverfindl** [comment]
Hello @BarryCarlyon,

seems this change broke `language` query string parameter in [/streams#get-live-streams](https://dev.twitch.tv/docs/v5/reference/streams#get-live-streams) endpoint.

In old version there could be defined multiple languages, separated by comma like on other kraken endpoints. Now it's possible to have only one language defined (not an actual issue in my case), but it was not pointed out in [announcement](https://discuss.dev.twitch.tv/t/upcoming-changes-to-the-streams-v5-endpoints/23926).

Real issue for me is, that everytime I try to set language parameter (single language, e.g.: ru), it returns always same unfiltered results.

Example:
```bash
$ curl -H 'Accept: application/vnd.twitchtv.v5+json' -H 'Client-ID: CLIENT_ID' -X GET 'https://api.twitch.tv/kraken/streams/?language=ru&limit=1'
```

Result:
```json
{
  "streams": [
    {
      "_id": 36992289488,
      "game": "League of Legends",
      "broadcast_platform": "live",
      "community_id": "",
      "community_ids": [],
      "viewers": 47154,
      "video_height": 1080,
      "average_fps": 59,
      "delay": 0,
      "created_at": "2020-02-22T07:30:47Z",
      "is_playlist": false,
      "stream_type": "live",
      "preview": {
        "small": "https://static-cdn.jtvnw.net/previews-ttv/live_user_lck-80x45.jpg",
        "medium": "https://static-cdn.jtvnw.net/previews-ttv/live_user_lck-320x180.jpg",
        "large": "https://static-cdn.jtvnw.net/previews-ttv/live_user_lck-640x360.jpg",
        "template": "https://static-cdn.jtvnw.net/previews-ttv/live_user_lck-{width}x{height}.jpg"
      },
      "channel": {
        "mature": false,
        "status": "HLE vs. GRF - APK vs. T1 [2020 LCK Spring Split]",
        "broadcaster_language": "en",
        "broadcaster_software": "",
        "display_name": "LCK",
        "game": "League of Legends",
        "language": "en",
        "_id": 124425501,
        "name": "lck",
        "created_at": "2016-05-16T18:35:22.08304Z",
        "updated_at": "2020-02-22T09:40:48.238827Z",
        "partner": true,
        "logo": "https://static-cdn.jtvnw.net/jtv_user_pictures/9c06645c-0ba0-4f53-8a41-8208a0e36043-profile_image-300x300.png",
        "video_banner": "https://static-cdn.jtvnw.net/jtv_user_pictures/f3120beb-66a6-4467-b2c6-89de0a8627df-channel_offline_image-1920x1080.jpeg",
        "profile_banner": "https://static-cdn.jtvnw.net/jtv_user_pictures/dbb0ef2a-cef9-4edf-b8ef-1ca1d1744978-profile_banner-480.jpeg",
        "profile_banner_background_color": "",
        "url": "https://www.twitch.tv/lck",
        "views": 66680527,
        "followers": 697329,
        "broadcaster_type": "",
        "description": "For more information, schedules and stats head to lolesports.com",
        "private_video": false,
        "privacy_options_enabled": false
      }
    }
  ]
}
```

Can you look into this?

Thanks

**Comment by BarryCarlyon** [comment]
@oliverfindl N,o I'm not staff.

The language issue is a separate problem and should be raised as a separate GitHub ticket/issue

This issue is a Documentation issue

**Comment by oliverfindl** [comment]
Hello @BarryCarlyon,

I submitted this as new [issue #66](https://github.com/twitchdev/issues/issues/66).

I think, it's also documentation issue, because docs are referencing to something, that's at this moment bugged/not working/not implemented without any notice about it.

Thanks

**Comment by jbulava** [comment]
There is a backlog ticket to update the docs for this update. It's internally on DXDOC-217.

**Comment by jbulava** [comment]
Docs were updated on Feb 25. The questions regarding the language parameter will be resolved on #66.
